localization/english/wc_buildings_l_english.yml

```diff
@@ -269,3 +275,632 @@
 
  building_quilboar_thorns_08:0 "Thorn Labyrinth"
  building_quilboar_thorns_08_desc:1 "This county has been reshaped to a Thorn Labyrinth of Agamaggan, the great boar god who died in battle against the Burning Legion. The Thorn Labyrinth is a massive and intricate network of thorny walls, towers, and tunnels, that cover the entire county. It is a formidable and impenetrable fortress for the quilboar, where they can defend themselves from any invasion or attack."
+ 
+ building_type_gearshaft_university:0 "Gearshaft University"
+ building_type_gearshaft_university_desc:0 "$building_type_generic_university_desc$\n\nThe Gearshaft University is a place for all gnomes to go and learn."
```

Comment:
So can only gnomes build it/does it get deactivated if non-gnomes hold it? Otherwise better get rid of the gnomes per se. Or make it that they are the ones who built it.